spec/metrics/relation_spec.rb

```diff
@@ -237,6 +257,26 @@
         end
       end
     end
+
+    context "complex queries" do
+      it "group + where" do
+        expect(rel.count('user_id').group(:traffic_source).fill(0).where(user_id: 123).past('28d').to_sql)
+          .to eq "select count(user_id) from \"dummy\" where (user_id=123) and (time > now() - 28d) " \
+                 "group by traffic_source fill(0)"
+      end
+
+      it "where + group + order + limit" do
+        expect(rel.group(:user_id).where(account_id: 123).order(account_id: :desc).limit(10).offset(10).to_sql)
+          .to eq "select * from \"dummy\" where (account_id=123) group by user_id " \
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.